clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -498,16 +499,22 @@ private UpdateMetadataRequest createUpdateMetadataRequest(int version, String ra
         partitionStates.put(new TopicPartition("topic20", 1),
                 new PartitionState(1, 0, 1, new ArrayList<>(isr), 2, new HashSet<>(replicas)));
 
-        Map<SecurityProtocol, UpdateMetadataRequest.EndPoint> endPoints1 = new HashMap<>();
-        endPoints1.put(SecurityProtocol.PLAINTEXT, new UpdateMetadataRequest.EndPoint("host1", 1223));
+        SecurityProtocol plaintext = SecurityProtocol.PLAINTEXT;
```

Comment:
Awkwardly placed comment: I think you are not actually testing V3...

i.e. I'd expect to see some test that calls createUpdateMetadataRequest(3,"rack") or something similar.